[google_maps_flutter] Reword README (#3784)
flutter/plugins: packages/google_maps_flutter/google_maps_flutter/README.md

  * To enable Google Maps for iOS, select "Maps SDK for iOS" in the "Additional APIs" section, then select "ENABLE".
  * Make sure the APIs you enabled are under the "Enabled APIs" section.

> * You can also find detailed steps on how to get started with the Google Maps Platform [here](https://developers.google.com/maps/gmp-get-started).

Reviewer: ```suggestion
For more details, see [Getting started with Google Maps Platform](https://developers.google.com/maps/gmp-get-started).
```
